w03/d1/morning_exercise/main.js

```diff
@@ -1 +1,18 @@
 // Write a program that will `console.log` the numbers from 1 to 100. But for multiples of three `console.log` “Fizz” instead of the number and for the multiples of five `console.log` “Buzz”. For numbers which are multiples of both three and five `console.log` “FizzBuzz”.
+for (let i=1; i<=100; i++){
+  if (i % 3 == 0){
+    console.log("Fizz");
+
+  }
+  else if  (i % 5 == 0){
+    console.log("Buzz");
+
+  }
+ else if (i % 3 == 0 && i % 5 == 0){
+    console.log("FizzBuzz");
+
+  }
+else  {
+  console.log(i);
+}
+}
```

Comment:
Hi!
Good job getting through this exercise. My only feedback is to double check your indentation next time. It helps keep your code readable. 
Thanks!